packages/panzer/adapters-ioss/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+TRIBITS_SUBPACKAGE(AdaptersIOSS)
+
+# Optional IOSS dependency
+##########################
+IF(${PACKAGE_NAME}_ENABLE_SEACASIoss)
+   GLOBAL_SET(PANZER_HAVE_IOSS ON)
+ELSE()
+   GLOBAL_SET(PANZER_HAVE_IOSS OFF)
+ENDIF()
+
```

Comment:
Do we need this block of code? This adapter code has a mandatory dependency on IOSS so I think we could eliminate this flag.